core-plugins/widgets/Excel-batchsource.json

```diff
@@ -136,7 +136,7 @@
       "label": "Error Handling",
       "properties": [
         {
-          "widget-type": "select",
+          "widget-type": "hidden",
```

Comment:
forgot to mention, please remove mention of these from the documentation if they exist

Reply:
Fixed